### ISSUE TITLE
loss_bbox和loss_dfl都为0

### ISSUE THREAD

**Issue by sosgu** [AILab-CVC/YOLO-World#581]
我在终端执行
CUDA_VISIBLE_DEVICES=1 python tools/train.py configs/finetune_coco/yolo_world_v2_l_vlpan_bn_sgd_1e-3_40e_8gpus_finetune_coco.py --work-dir log --amp --resume
命令后，训练之后的显示
`style="text-decoration-style:single">INFO</u></font> - Epoch(train) [25][50/90]  base_lr: 1.0000e-03 lr: 4.3075e-04  eta: 0:09:11  time: 0.3959  data_time: 0.0126  memory: 5473  grad_norm: 0.0283  loss: 0.0016  loss_cls: 0.0016  loss_bbox: 0.0000  loss_dfl: 0.0000`
我在网上搜索相关的网页说要保持json文件里面标签要一直，但是我将其修改后还是这个效果，所以我想在这里咨询一下大家，这个问题应该怎么解决；
我在训练集中的类别为
    "categories": [
        {
            "id": 0,
            "name": "head",
            "supercategory": "head"
        }
    ]
在测试集中的类别为
    "categories": [
        {
            "id": 0,
            "name": "head",
            "supercategory": "head"
        }
    ]
coco_class_texts.json里面的内容为
[["head"]]

最后是yolo_world_v2_l_vlpan_bn_sgd_1e-3_40e_8gpus_finetune_coco.py文件中的为
metainfo = dict(classes=("head"))
coco_train_dataset = dict(_delete_=True,
                          type='MultiModalDataset',
                          dataset=dict(
                              type='YOLOv5CocoDataset',
                              metainfo=metainfo,
只截取了部分代码，还请各位大佬能够指导一下

**Comment by aoji0606** [comment]
同问

**Comment by danielgf0824** [comment]
same problem！

**Comment by chuanxiang-wu** [comment]
+1

**Comment by yuhang1112** [comment]
+1！

**Comment by 2950375033** [comment]
我也遇到了这个问题，请问你解决了吗

**Comment by chuanxiang-wu** [comment]
> 我也遇到了这个问题，请问你解决了吗

还没呢

**Comment by 2950375033** [comment]
> 同问

请问你找到解决方法了吗？

**Comment by fang35054-sudo** [comment]
请问解决了吗

**Comment by weilanShi** [comment]
> 我在终端执行 CUDA_VISIBLE_DEVICES=1 python tools/train.py configs/finetune_coco/yolo_world_v2_l_vlpan_bn_sgd_1e-3_40e_8gpus_finetune_coco.py --work-dir log --amp --resume 命令后，训练之后的显示 `style="text-decoration-style:single">INFO</u></font> - Epoch(train) [25][50/90] base_lr: 1.0000e-03 lr: 4.3075e-04 eta: 0:09:11 time: 0.3959 data_time: 0.0126 memory: 5473 grad_norm: 0.0283 loss: 0.0016 loss_cls: 0.0016 loss_bbox: 0.0000 loss_dfl: 0.0000` 我在网上搜索相关的网页说要保持json文件里面标签要一直，但是我将其修改后还是这个效果，所以我想在这里咨询一下大家，这个问题应该怎么解决； 我在训练集中的类别为 "categories": [ { "id": 0, "name": "head", "supercategory": "head" } ] 在测试集中的类别为 "categories": [ { "id": 0, "name": "head", "supercategory": "head" } ] coco_class_texts.json里面的内容为 [["head"]]
> 
> 最后是yolo_world_v2_l_vlpan_bn_sgd_1e-3_40e_8gpus_finetune_coco.py文件中的为 metainfo = dict(classes=("head")) coco_train_dataset = dict(_delete_=True, type='MultiModalDataset', dataset=dict( type='YOLOv5CocoDataset', metainfo=metainfo, 只截取了部分代码，还请各位大佬能够指导一下

标签里没有mask的配置文件不要使用那个mask的，直接使用base 配置文件，这两个就能正常
